code/src/components/Summary.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+const Summary = ({radioValue, dropDown, firstThought, societysPortrayal}) => {
+    return (
+        <div>
+            <h2>Your answers</h2>
+            <p>1: {firstThought}</p>
+            <p>2: {radioValue}</p>
+            <p>3: {dropDown}</p>
+            <p>4: {societysPortrayal}</p>
+        </div>
+    )
+}
+
+export default Summary;
```

Comment:
I would have liked to see the summery centered on the screen. Otherwise, very good job!